.github/workflows/ci/pkg/bot/check.go

```diff
@@ -57,6 +57,15 @@ func (c *Bot) checkInternal(ctx context.Context) error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
+	// If @r0mant and @russjones approve a pull request, allow the check workflow run to pass.
```

Comment:
It should be pretty easy to generalise this so that it's not fixed on the idea of there being exactly two admins. 

```go
// repoAdminsHaveApproved checks that *all* listed admins have approved the change. 
func repoAdminsHaveApproved() bool {
  // Don't want to accidentally approve if the admins aren't set
  if len(ci.RepoAdmins) == 0 {
    return false
  }

  for _, adminName := range ci.RepoAdmins {
    if !hasApproved(adminName, mostRecentReviews) {
      return false
    }
  }

  return true
}
```

Reply:
+1, this would be a nice change.

In addition - is it possible to get the list of approvals in a single API call and then just check for membership in that list?

Reply:
If you're worried about the number of API calls we are making, `hasApproved` doesn't make an API call for each call to it rather `getMostRecentReviews` makes a call for all the reviews and stores the reviews as our own type (which is then passed in to `hasApproved`). I believe there is a way check what membership a user has in relation to the Teleport repo, but I believe that would be another API call.

Reply:
@quinqu It's more as we add more reviewers, you're going to get a really long and complex `if` statement.

Also, please make this an `OR` instead of an `AND`.